src/diagnostics.toit

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Toitware ApS. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be found
+// in the LICENSE file.
+
+/**
+Diagnostics information for a GNSS receiver.
+*/
+class GnssDiagnostics:
+  /**
+  The time to first fix.
+  0, if no fix has been obtained.
+  */
+  time_to_first_fix/Duration
+
+  // TODO(Lau): Detail this measure.
```

Comment:
Perhaps we should just omit this part, until we learn how GNSS signal quality is really computed?

Reply:
We compute the signal quality (and it is the average of the cno of the first 4 satellites).